### ISSUE TITLE
Trying to install pyne from anaconda, but apparently it doesn't exist?

### ISSUE THREAD

**Issue by mcgeochd** [pyne/pyne#1245]
Following the guidance here https://pyne.io/install/conda.html#conda, or even just trying

conda create -n test pyne

doesn't work. If its no longer in conda-forge I would understand that, but according to the installation instructions it is, so I'm a little confused as to whats happened.

Running conda search pyne generates the following log:
[pyne search.txt](https://github.com/pyne/pyne/files/5013131/pyne.search.txt)

**Comment by gonuke** [comment]
It looks like you have been trying to install PyNE on Windows using anaconda.  Unfortunately, PyNE is not supported on Windows at this time - only on Mac or Linux.

**Comment by mcgeochd** [comment]
Thanks for the heads up, is there an equivalent package for windows that you know of?

**Comment by gonuke** [comment]
Which features of PyNE are you trying to access?

**Comment by mcgeochd** [comment]
I'm not certain at this present moment. I had hoped to used PyNE to model the performance of an arbitrary nuclear thermal rocket, alongside cantera, thermo, and fluids, but have not yet advanced to the point where I could determine what features would be necessary for that.

**Comment by mcgeochd** [comment]
> It looks like you have been trying to install PyNE on Windows using anaconda. Unfortunately, PyNE is not supported on Windows at this time - only on Mac or Linux.

This text: "Conda Package Manager - The conda package manager is ideal for PyNE users on Linux, Mac OSX, and Windows." should probably be ammended in that case to bring it line with other text on the page that excludes windows https://pyne.io/install/index.html

**Comment by cjwyett** [comment]
Have you tried the PyNE VirtualBox? Not an ideal solution, but if you're looking to have a play around...

**Comment by gonuke** [comment]
I agree @mcgeochd - we are planning to spend some time overhauling the documentation this month

**Comment by mcgeochd** [comment]
> Have you tried the PyNE VirtualBox? Not an ideal solution, but if you're looking to have a play around...

I haven't but I will now, that would only be a stopgap though as I would prefer to write a full program myself without the need to bounce between that and a 3rd party solution.

**Comment by gonuke** [comment]
You may also be interested in ARMI: https://github.com/terrapower/armi
